addon/adapters/osf-adapter.js

```diff
@@ -42,8 +43,20 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
             var serializer = store.serializerFor(relationType);
             if (relationMeta.kind === 'hasMany') {
                 // A hack, since we'd have to use a bulk requests to send a list; TODO remove [0]
-                serialized = snapshot.hasMany(relationship).filter(record => record.id === null).map(record => serializer.serialize(record))[0];
-                delete serialized.data.relationships;
+                serialized = snapshot.hasMany(relationship).filter(record => record.id === null).map(record => serializer.serialize(record));
+                if (serialized.length > 1) {
+                    this.isBulk = true;
+                    serialized = {
+                        data: serialized.map(function(record) {
+                            var data = record.data;
+                            delete data.relationships;
+                            return data;
```

Comment:
Thinking ahead, do we intend to support other kinds of bulk requests (besides relationships)?

A typical ember pattern is to offer different methods for each behavior- kind of like how serializers offer `normalizeSingleResponse` vs `normalizeArrayResponse`. Looking at this `_relationshipPayload` method, I think we should do something similar for bulk requests, so that it is possible to override single or bulk behavior independently, without having to change both.